cvat-canvas/src/typescript/canvasView.ts

```diff
@@ -875,7 +886,22 @@ export class CanvasViewImpl implements CanvasView, Listener {
                             'stroke-width': consts.POINTS_STROKE_WIDTH / getGeometry().scale,
                         });
 
-                    circle.on('mouseenter', (): void => {
+                    circle.on('mouseenter', (e: MouseEvent): void => {
+                        const activeElement = getActiveElement();
+                        if (activeElement !== null && e.altKey) {
+                            const [state] = getController().objects.filter(
+                                (_state: any): boolean => _state.clientID === activeElement.clientID,
+                            );
+                            if (state.shapeType === 'points') {
```

Comment:
```suggestion
                            if (state?.shapeType === 'points') {
```

Reply:
Maybe this condition should work for all the shapes, not only for points, do not you think so?